src/pages/app/p/[p]/bundle/[bundle].vue

```diff
@@ -32,6 +35,31 @@ const channel = ref<(Database['public']['Tables']['channels']['Row'])>()
 const version_meta = ref<Database['public']['Tables']['app_versions_meta']['Row']>()
 const secondaryChannel = ref<boolean>(false)
 
+const showChannel = computed(() => {
+  // channel.value && ((!channel.value.enableAbTesting && !channel.value.enable_progressive_deploy && channel.value.version === version.value.id) || ((channel.value.enableAbTesting || channel.value.enable_progressive_deploy) && channel.value.secondVersion === version.value.id))
+  if (!channel.value || !version.value)
+    return false
```

Comment:
we really need this? Since it'll return false on the end anyways
```suggestion
```

Reply:
if you're doing this to show in the future that this condition should not show the channel, feel free to keep it as it is